docs/EnterpriseScaleAnalytics-Prerequisites.md

```diff
@@ -1,6 +1,6 @@
 # Data Management Zone - Prerequisites
 
-This template repsitory contains all templates to deploy the Data Management Zone of the Enterprise-Scale Analytics architecture. The Data Management Zone is the central management instance to govern all data assets across all Data Landing Zones and possible even beyond that.
+This template repsitory contains all templates to deploy the Data Management Zone of the Data Management & Analytics architecture. The Data Management Zone is the central management instance to govern all data assets across all Data Landing Zones and possible even beyond that.
```

Comment:
Typo error "repsitory" -> "repository".

Consider reframing the sentence? In my opinion, It's clear that DMZ is part of DMA so we don't need to repeat it. 
Perhaps include hyperlink to Data Management Zone so that we do not need to describe it again to the readers?

"This repository contains all of the templates to deploy Data Management Zone. Data Management Zone is a set of services to govern the data assets across various Data Landing Zones associated with it."